src/useTimer.ts

```diff
@@ -66,12 +62,12 @@ export const useTimer = (config?: IConfig): IValues => {
   };
 
   const start = () => {
-    if (intervalRef.current) {
-      return;
+    if (intervalRef.current === null) {
```

Comment:
With truthiness, no need to explicitly check for null value. And i prefer to short circuit condition by returning first, and putting logic outside if block.